apiary.apib

```diff
@@ -2618,6 +2618,124 @@ Your Accredible account is represented as an Issuer within the API.
             }
         }
 
+# Group Data Exports
+
+## Data Exports [/v1/data_exports]
+
+### View All Data Exports [GET]
+
+Returns list of data exports of departments accessible via the used API_KEY.
+
++ Request (application/json)
+
+    + Headers
+            Authorization: Token token=YOUR API_KEY
+
++ Response 200 (application/json)
```

Comment:
if you look through this documentation you will find that we don't have any `#index` endpoints for other objects (except deprecated ones). Which makes sense as a listing could be quite huge after a period of time.

Should this endpoint not be a `POST /v1/data_exports/search` ?

Reply:
or add it to the list of deprecated `#index` endpoints